lib/elixir/lib/system.ex

```diff
@@ -44,7 +44,7 @@ defmodule System do
       time may not match in case of time warps although the VM works towards
       aligning them. This time is not monotonic (i.e., it may decrease)
       as its behaviour is configured [by the VM time warp
-      mode](http://erlang.org/doc/apps/erts/time_correction.html#Time_Warp_Modes);
+      mode](http://www.erlang.org/doc/apps/erts/time_correction.html#Time_Warp_Modes);
```

Comment:
yes, we do need it.
erlang.org front page redirects, not every page in the server.
All documentation links point to www, (and this is the only one that does not), .. you will force people to reload all cached files by linking to a different subdomain